src/Microsoft.Health.Abstractions/Microsoft.Health.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
>1 [](start = 34, length = 1)

nit: Is this necessary?

Reply:
I think I'll revert this. In https://docs.microsoft.com/en-us/dotnet/standard/net-standard .net5 implements .net standard 2.1, .net 4.6.1 doesn't.
So we should keep 2.0 until we move it to .net5.

Reply:
I'm just curious, does anyone consume Abstraction without other projects we have in this repo?

Reply:
No, I wouldn't think so